### PR TITLE
Suggestion for Go build version and Staticcheck and Commands Added

### PR DESCRIPTION
**Pull request by Shivam010**
Suggestion for new Go build version in travis and Code Reviewer
-
- Removing go 1.9.x and tip
- Adding go 1.11.x
- Staticcheck offers extensive and customizable code analysis and review. It would be best for the job.
- Command **JSON.OBJKEYS** added: https://oss.redislabs.com/rejson/commands/#jsonobjkeys 
- Command **JSON.OBJLEN** added: https://oss.redislabs.com/rejson/commands/#jsonobjlen
- Command **JSON.DEBUG** added: https://oss.redislabs.com/rejson/commands/#jsondebug

Fix for issue: #10, #11, #12, #16

### CODE REVIEW

File: rejson.go [hunk 1]
```diff
@@ -3,30 +3,43 @@ package rejson
...
 	"github.com/gomodule/redigo/redis"
 )
 
-// PopArrLast gives index of the last element for JSONArrPop
-const PopArrLast = -1
+const (
+	// PopArrLast gives index of the last element for JSONArrPop
+	PopArrLast = -1
+	// DebugMemorySubcommand & DebugHelpSubcommand provide the corresponding sub commands for JSONDebug
+	DebugMemorySubcommand = "MEMORY"
```
Comment (nitishm) [inline rejson.go]: Can you add a comment for this const

File: rejson.go [hunk 2]
```diff
@@ -3,30 +3,43 @@ package rejson
...
+	DebugHelpSubcommand   = "HELP"
```
Comment (nitishm) [inline rejson.go]: Can you add a comment for this const

Comment (Shivam010) [reply rejson.go]: The comments for `DebugMemorySubcommand` & `DebugHelpSubcommand` are combined to a single comment.

**Comment by Shivam010** [comment]
Sorry @nitishm, I forgot to mention the updates on issues #13 and #14
- Command already **JSON.RESP** added: https://oss.redislabs.com/rejson/commands/#jsonresp
- Command already **JSON.FORGET** added: https://oss.redislabs.com/rejson/commands/#jsonforget

Fixed #13 #14

**Pull request merged**